gene expression between samples was calculated through the inclusion of a cDNA dilution series of a reference sample, amplified with each primer set separately. In order to account for RNA loading differences between samples, the measured gene expression was normalized against β-actin expression. The t-test was used to assign statistical significance.

NF-κB reporter system
The ability of DCA to activate NF-κB was investigated using a luciferase reporter system in which the luciferase gene was under the control of a NF-κB dependent promoter (Stratagene, Cambridge, UK). In short, the OE33 cells were seeded in triplicate into 6 well tissue culture flasks and the NF-κB luciferase plasmid was transfected into them using the genejammer transfection reagent (Stratagene). Initial experiments validated the transfection using a supplied positive control for NF-κB activity. The cells were then either exposed to DCA for varying periods of time (0–4 h) or remained untreated. After exposure, the cells were processed following the manufacturer’s recommendations to release the luciferase enzyme and the subsequent luciferase expression was quantified with the Bright-Glo luciferase assay system (Promega, Southampton, UK). Resulting luminescence was measured using a luminometer (Anthos Labtech Instruments, Salzburg, Austria). Fold increases were calculated based on the background luminescence of the untreated cells (untreated cells were set up for each time point analysed). The t-test was used to evaluate differences between the triplicate readings from treated and untreated cells.

Results

DCA specifically switches on IκB and IL-8
RNA extracted from untreated OE33 cells and OE33 cells treated for 24 h with 300 μM DCA at pH 7, was analysed using five different pathway specific membrane arrays. By far the most prominent of the DCA induced gene expression changes were the switching on of IκB and IL-8. As IκB and IL-8 are transcriptional targets of NF-κB, this suggests activation of NF-κB by DCA (as is demonstrated below). As shown in Figure 2, using a cancer pathway array and a signal transduction array, both IκB and IL-8 are substantially up-regulated 24 h after DCA treatment. Figure 2 also shows a number of other changes in gene expression 24 h after DCA treatment (mostly up-regulation), these (and other changes noted from other arrays) are documented in Table I.

The gene expression alterations listed in Table I from the five different arrays used, have been grouped into four separate groups based on the function of the genes involved. As IL-8 (29) and IκB (30) have previously been suggested to be transcriptional

<table>
  <tr>
    <th>Gene</th>
    <th></th>
    <th>Time point</th>
  </tr>
  <tr>
    <td colspan="3">NFκB related genes</td>
  </tr>
  <tr>
    <td>Tumour necrosis factor alpha induced protein (TNFaip)</td>
    <td>↑↑</td>
    <td>4–24 h</td>
  </tr>
  <tr>
    <td>Flice inhibitory protein (FLIP), Cflar, Casper</td>
    <td>↑</td>
    <td>24 h</td>
  </tr>
  <tr>
    <td colspan="3">Cellular proteases/invasion genes</td>
  </tr>
  <tr>
    <td>Plasminogen activator, Urokinase (Plau)</td>
    <td>↑↑↑</td>
    <td>1–24 h</td>
  </tr>
  <tr>
    <td>Plasminogen activator receptor (PlauR)</td>
    <td>↑↑</td>
    <td>24 h</td>
  </tr>
  <tr>
    <td>Plasminogen activator inhibitor 1 (Serpine 1)</td>
    <td>↓↓</td>
    <td>24 h</td>
  </tr>
  <tr>
    <td>Serine protease inhibitor B5 (maspin, Serpine B5)</td>
    <td>↑↑</td>
    <td>24 h</td>
  </tr>
  <tr>
    <td>Plasminogen activator inhibitor B2 (Serpine B2, PAI-2)</td>
    <td>↑↑</td>
    <td>24 h</td>
  </tr>
  <tr>
    <td colspan="3">Growth factors/receptors</td>
  </tr>
  <tr>
    <td>Melanoma growth stimulating activity (MGSA, Gro 1)</td>
    <td>↑</td>
    <td>24 h</td>
  </tr>
  <tr>
    <td>Vascular endothelial growth factor (VEGF)</td>
    <td>↑↑</td>
    <td>4–24 h</td>
  </tr>
  <tr>
    <td>Epidermal growth factor receptor (EGFR)</td>
    <td>↑</td>
    <td>1–24 h</td>
  </tr>
  <tr>
    <td colspan="3">Adhesion molecules</td>
  </tr>
  <tr>
    <td>β-Catenin</td>
    <td>↑</td>
    <td>1–24 h</td>
  </tr>
  <tr>
    <td>Integrin B1, integrin A6</td>
    <td>↑</td>
    <td>4–24 h</td>
  </tr>
  <tr>
    <td>E-Cadherin</td>
    <td>↑</td>
    <td>1 h</td>
  </tr>
  <tr>
    <td>Intercellular adhesion molecule (Icam)</td>
    <td>↑↑</td>
    <td>4–24 h</td>
  </tr>
  <tr>
    <td>Neural cell adhesion molecule (Ncam)</td>
    <td>↑↑</td>
    <td>1–4 h</td>
  </tr>
</table>

This table contains a list of genes whose expression was consistently altered after exposure to DCA. By consistent, we mean that the expression change was evident on both duplicate array experiments (although no real-time PCR data available). In the right hand column is the time point at which the expression change was evident. The arrows in the central column reflect the relative strength of the expression changes.

![Membrane array analysis of DCA treated OE33 cells, 24 h post-exposure. (a) Cancer pathway arrays were used to study gene expression abnormalities induced by 300 μM DCA relative to untreated OE33 cells. These example arrays show the up-regulation of the IκB and IL-8 genes by DCA. (b) Two example signal transduction arrays are shown; those hybridized with cDNA from untreated OE33 cells and cDNA from OE33 cells 24 h after DCA exposure. A number of other genes are up-regulated by DCA, particularly evident in the cancer pathway array in (a). These genes (and others) are identified in Table I.](page_1012_1342_484_388.png)

Fig. 2. Membrane array analysis of DCA treated OE33 cells, 24 h post-exposure. (a) Cancer pathway arrays were used to study gene expression abnormalities induced by 300 μM DCA relative to untreated OE33 cells. These example arrays show the up-regulation of the IκB and IL-8 genes by DCA. (b) Two example signal transduction arrays are shown; those hybridized with cDNA from untreated OE33 cells and cDNA from OE33 cells 24 h after DCA exposure. A number of other genes are up-regulated by DCA, particularly evident in the cancer pathway array in (a). These genes (and others) are identified in Table I.