Table 1. Bacterial strains

<table>
  <tr>
    <th>Strain</th>
    <th>Relevant characteristics</th>
    <th>Source or reference</th>
  </tr>
  <tr>
    <td>W3110</td>
    <td>wild type</td>
    <td>Laboratory stock</td>
  </tr>
  <tr>
    <td>JE7951</td>
    <td>spr-1</td>
    <td>(7)</td>
  </tr>
  <tr>
    <td>JE7934</td>
    <td>preA::neo</td>
    <td>(7)</td>
  </tr>
  <tr>
    <td>JE7456</td>
    <td>str xyl argG</td>
    <td>Laboratory stock</td>
  </tr>
  <tr>
    <td>CAG12072</td>
    <td>zgi-203::Tn10</td>
    <td>(26)</td>
  </tr>
  <tr>
    <td>TOP10</td>
    <td>recA araD139 ΔaraA-leu) 7697</td>
    <td>Invitrogen</td>
  </tr>
  <tr>
    <td>Rev1</td>
    <td>spr-1 nplI1</td>
    <td>Spontaneously from JE7951</td>
  </tr>
  <tr>
    <td>Rev2</td>
    <td>spr-1 nplI2</td>
    <td>Spontaneously from JE7951</td>
  </tr>
  <tr>
    <td>Rev3</td>
    <td>spr-1 nplI3</td>
    <td>Spontaneously from JE7951</td>
  </tr>
  <tr>
    <td>Rev4</td>
    <td>spr-1 nplI4</td>
    <td>Spontaneously from JE7951</td>
  </tr>
  <tr>
    <td>Rev5</td>
    <td>spr-1 nplI5</td>
    <td>Spontaneously from JE7951</td>
  </tr>
  <tr>
    <td>Rev6</td>
    <td>spr-1 nplI6</td>
    <td>Spontaneously from JE7951</td>
  </tr>
  <tr>
    <td>TAR1</td>
    <td>str xyl argG zgi-203::Tn10</td>
    <td>CAG12072 (P1) × JE7456</td>
  </tr>
  <tr>
    <td>TAR2</td>
    <td>argG zgi-203::Tn10</td>
    <td>TAR1 (P1) × W3110</td>
  </tr>
  <tr>
    <td>TAR3</td>
    <td>spr-1 argG zgi-203::Tn10</td>
    <td>TAR1 (P1) × JE7951</td>
  </tr>
  <tr>
    <td>KM354</td>
    <td>recJ pTP223 gam bet exo</td>
    <td>(14)</td>
  </tr>
  <tr>
    <td>MU2</td>
    <td>nplI::cm</td>
    <td>Derived from KM354 by transformation with linearized nplI::cm DNA</td>
  </tr>
  <tr>
    <td>MU24</td>
    <td>nplI::cm</td>
    <td>MU2 (P1) × W3110</td>
  </tr>
  <tr>
    <td>MU25</td>
    <td>nplI1</td>
    <td>Rev1 (P1) × TAR2</td>
  </tr>
  <tr>
    <td>MU26</td>
    <td>nplI2</td>
    <td>Rev2 (P1) × TAR2</td>
  </tr>
  <tr>
    <td>MU27</td>
    <td>nplI3</td>
    <td>Rev3 (P1) × TAR2</td>
  </tr>
  <tr>
    <td>MU124</td>
    <td>spr-1 nplI::cm</td>
    <td>MU2 (P1) × JE7951</td>
  </tr>
  <tr>
    <td>MU125</td>
    <td>spr-1 nplI1</td>
    <td>Rev1 (P1) × TAR3</td>
  </tr>
  <tr>
    <td>MU126</td>
    <td>spr-1 nplI2</td>
    <td>Rev2 (P1) × TAR3</td>
  </tr>
  <tr>
    <td>MU127</td>
    <td>spr-1 nplI3</td>
    <td>Rev3 (P1) × TAR3</td>
  </tr>
  <tr>
    <td>PN1</td>
    <td>preA::neo nplI::cm</td>
    <td>MU24 (P1) × JE7934</td>
  </tr>
  <tr>
    <td>HSB1</td>
    <td>ibpB::cm</td>
    <td>Derived from KM354 by transformation with linearized ibpB::cm DNA</td>
  </tr>
  <tr>
    <td>HSB2</td>
    <td>ibpB::cm</td>
    <td>HSB1 (P1) × W3110</td>
  </tr>
  <tr>
    <td>HSB3</td>
    <td>spr-1 ibpB::cm</td>
    <td>HSB1 (P1) × JE7951</td>
  </tr>
</table>

(underlined), 5'-TGGGCCATGGAGCCTTTTTTGCG-3' and 5'-GTCACCATGGGCTGCTGCCCTTCTGCC-3'. The amplified DNA was digested with NcoI, then inserted into the same site of pBAD Myc-His C. To construct pN16 encoding the hexa-histidine-tagged NplI with the amino-acid substitution G103D, the mutated nplI in the chromosomal DNA of Rev6 (spr-1 nplI6) was amplified by PCR using the primers described above, and the amplified DNA was digested with the NcoI and inserted into the same site of pBAD Myc-His C. To construct pN17 encoding the truncated and hexa-histidine-tagged NplI, the fragment containing the truncated nplI was amplified by PCR using the forward primer 5'-TGGGCCATGGAGCCTTTTTTGCG-3' and the reverse primer 5'-GGTCCATGGGGCTTGGCCC-3'. The amplified DNA was digested with NcoI, then inserted into the same site of pBAD Myc-His C. pN18 and pN19 were constructed in the same way as pN17 except that 5'-CATCCATGGGCCCTGGCCCAAGG-3' or 5'-CTTCCATGGGGCCCAAGGAGG-3', respectively, was used as the reverse primer in PCR. To construct pN110 encoding NplI, the nplI in the chromosomal DNA of W3110 was amplified by PCR using the forward primer containing the NcoI site (underlined) 5'-TGGGCCATGGAGCCTTTTTTGCG-3' and the reverse primer containing the BamHI site (underlined) 5'-TGGATCCGCTCAAAGTAGG-3'. The amplified DNA was digested with NcoI and BamHI and inserted into the same sites of pTrc99A. pN112, pN113, and pN114 were constructed in the same way as pN110 except that 5'-CCAGGATCCCTTAGTCCTGGCC-3', 5'-GTCAGGATCCTACTGGCCCAAGG-3', or 5'-ATCTGGATCCTAGCCCAAGGAG-3', respectively, was used as the reverse primer containing the BamHI site (underlined) in PCR. The amplified DNA was digested with NcoI and BamHI and then inserted into the same sites of pTrc99A.

Expression and Purification of Hexa-Histidine-Tagged Npl Protein.—E. coli cells were transformed with the recombinant plasmids. The transformants were grown in L broth at 42°C for 4 h, then incubated in L broth containing 0.01% arabinose at 42°C for 4 h for expression of the fusion genes. Cells from 400 ml of medium were collected, washed once with ice-cold 50 mM sodium phosphate buffer (pH 8.0) containing 300 mM NaCl (buffer A), suspended in 4 ml of the buffer A containing 10 mM imidazole and 1% Triton X-100, and disrupted by sonication with cooling by ice-water. The mixture was centrifuged for 15 min at 17,700 ×g to obtain a clear lysate. Four milliliters of the supernatant was mixed with 1 ml of nickel-nitrilotriacetic acid (Ni-NTA) resin and incubated with rotation for 2 h at 4°C. The mixture was loaded onto an open column. The column was washed with 4 ml each of buffer A containing 20 mM imidazole and then containing 40 mM imidazole. The bound proteins were eluted with 1.2 ml of buffer A containing 100 mM imidazole, and fractions of 0.4 ml were collected.

Mass Spectrometry Analysis.—An electrospray ion trap mass spectrometer (LCQ DECA XP ion trap mass system, Thermo Electron) coupled online with a capillary HPLC (Magic 2002, Michrom BioResources) was used to acquire MS/MS spectra. A 0.2 × 50 mm MAGICMS C18