TABLE 3. Mapping of transposon insertions sites that result in white phenotype in C. sakazakii ES5

<table>
  <tr>
    <th>COG functional category<sup>a</sup></th>
    <th>COG functional class</th>
    <th colspan="2">Annotation<sup>b,c</sup></th>
  </tr>
  <tr>
    <th></th>
    <th></th>
    <th>Homologue</th>
    <th>Gene product</th>
  </tr>
  <tr>
    <td rowspan="6">Mutation in pigment operon<br>H: coenzyme transport and metabolism<br>GC: carbohydrate transport and metabolism/signal transduction mechanisms<br>R/E: general function prediction only/amino acid transport and metabolism<br>Q: secondary metabolites biosynthesis, transport and catabolism<br>I: lipid transport and metabolism</td>
    <td>Geranylgeranyl pyrophosphate synthase</td>
    <td>crtE</td>
    <td>Geranylgeranyl pyrophosphate synthase</td>
  </tr>
  <tr>
    <td>Glucosyl transferases, related to UDP-glucosyltransferase</td>
    <td>crtX</td>
    <td>Zeaxanthin glucosyl transferase</td>
  </tr>
  <tr>
    <td>Acetyltransferase/choline dehydrogenase and related flavoproteins</td>
    <td>crtY</td>
    <td>Lycopene cyclase</td>
  </tr>
  <tr>
    <td>Phytoene dehydrogenase and related proteins</td>
    <td>crtI</td>
    <td>Phytoene dehydrogenase</td>
  </tr>
  <tr>
    <td>Phytoene/squalene synthase</td>
    <td>crtB</td>
    <td>Phytoene synthase</td>
  </tr>
  <tr>
    <td colspan="4"></td>
  </tr>
  <tr>
    <td>Mutation outside pigment operon<br>C: energy production and conversion</td>
    <td>F<sub>o</sub>F<sub>i</sub>-type ATP synthase, subunit alpha<br>F<sub>o</sub>F<sub>i</sub>-type ATP synthase, subunit beta<br>F<sub>o</sub>F<sub>i</sub>-type ATP synthase, subunit gamma<br>F<sub>o</sub>F<sub>i</sub>-type ATP synthase, subunit epsilon (mitochondrial delta subunit)<br>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E1) component, and related enzymes<br>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E2) component, and related enzymes<br>Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E3) component, and related enzymes<br>Malate/lactate dehydrogenases<br>Succinate dehydrogenase/fumarate reductase, flavoprotein subunit<br>Na<sup>+</sup>/H<sup>+</sup> antiporter</td>
    <td>ESA_04012<br>ESA_04006<br>ESA_04007<br>ESA_04005<br>ESA_02622<br>ESA_02621<br>ESA_03222<br>ESA_03622<br>ESA_02624<br>ESA_03316</td>
    <td>F<sub>o</sub>F<sub>i</sub> ATP synthase subunit alpha<br>F<sub>o</sub>F<sub>i</sub> ATP synthase subunit beta<br>F<sub>o</sub>F<sub>i</sub> ATP synthase subunit gamma<br>F<sub>o</sub>F<sub>i</sub> ATP synthase subunit epsilon<br>sucA 2-oxoglutarate dehydrogenase E1 component<br>Dihydrolipoamide acetyltransferase<br>aceF dihydrolipoamide acetyltransferase<br>Malate dehydrogenase<br>Succinate dehydrogenase<br>flavoprotein subunit<br>pH-dependent sodium/proton antiporter<br>cAMP regulatory protein</td>
  </tr>
  <tr>
    <td>P: inorganic ion transport and metabolism<br>T: signal transduction mechanisms</td>
    <td>cAMP-binding proteins, catabolite gene activator, and regulatory subunit of cAMP-dependent protein kinases<br>DnaK suppressor protein</td>
    <td>ESA_04376<br>ESA_03194</td>
    <td>cAMP regulatory protein<br>DnaK transcriptional regulator DksA</td>
  </tr>
  <tr>
    <td>S: function unknown</td>
    <td>Uncharacterized conserved protein</td>
    <td>ESA_04343 (Ent638_3811)<sup>d</sup><br>ESA_03563 (ETA_03450)<sup>d</sup><br>ESA_00549 (AAG53883)<sup>d</sup></td>
    <td>Hypothetical protein (intracellular growth attenuator IgA, Enterobacter sp. 638)<sup>d</sup><br>Hypothetical protein (YhbC-like protein, Erwinia tasmaniensis Et1/99)<sup>d</sup><br>Hypothetical protein (sigma factor RpoS, Escherichia coli)<sup>d</sup></td>
  </tr>
</table>

<sup>a</sup> NCBI clusters of orthologous groups (COG) of proteins.
<sup>b</sup> Cronobacter sakazakii ES5 BAC 9E10 for mutations within the pigment operon (accession no. AM384990.1).
<sup>c</sup> NCBI assembly ATCC BAA-894 C. sakazakii complete genome for mutations outside pigment operon (accession no. CP000783.1).
<sup>d</sup> Closest annotated homolog.

(data not shown), consistently with the results for LB, all mutant strains showed significantly increased maximum rates of growth (\( \mu_{\text{max},w} = 0.14; \mu_{\text{max},\Delta crtX} = 0.22; \mu_{\text{max},\Delta crtE} = 0.24; \) and \( \mu_{\text{max},\Delta crtY} = 0.19; P = 0.000 \)).

Cold stress experiments were performed by growing \( \Delta crtE \), \( \Delta crtX \), and \( \Delta crtY \) in LB and M9 medium at 10Â°C (for results in M9, see Fig. 1B). Under these conditions, no significant differences in maximum specific growth rates were detected for \( \Delta crtY \), \( \Delta crtE \), and \( \Delta crtX \) in both LB and M9 compared to those of the wild type (in LB, \( \mu_{\text{max},w} = 0.04, \mu_{\text{max},\Delta crtX} = 0.03, \mu_{\text{max},\Delta crtE} = 0.03 \), and \( \mu_{\text{max},\Delta crtY} = 0.04 \); in M9, \( \mu_{\text{max},w} = 0.01, \mu_{\text{max},\Delta crtX} = 0.01, \mu_{\text{max},\Delta crtE} = 0.01 \), and \( \mu_{\text{max},\Delta crtY} = 0.01 \)).

To evaluate growth under acidic conditions, wild-type and