<|ref|>title<|/ref|><|det|>[[85, 107, 848, 164]]<|/det|>
# Y-Chromosome Variation in Southern African Khoe-San Populations Based on Whole-Genome Sequences  

<|ref|>text<|/ref|><|det|>[[85, 171, 815, 210]]<|/det|>
Thijessen Naidoo \(^{1,2,3,4,\dagger}\) , Jingzi Xu \(^{1,\dagger}\) , Mário Vicente \(^{1}\) , Helena Malmström \(^{1,5}\) , Himla Soodyal \(^{6,7,8}\) , Mattias Jakobsson \(^{1,3,5}\) , and Carina M. Schlebusch \(^{1,3,5,\star}\)  

<|ref|>text<|/ref|><|det|>[[85, 227, 900, 365]]<|/det|>
\(^{1}\) Human Evolution, Department of Organismal Biology, Evolutionary Biology Centre, Uppsala University, Sweden \(^{2}\) Department of Archaeology and Classical Studies, Stockholm University, Sweden \(^{3}\) Science for Life Laboratory, Uppsala, Sweden \(^{4}\) Centre for Palaeogenetics, Stockholm, Sweden \(^{5}\) Palaeo- Research Institute, University of Johannesburg, Auckland Park, South Africa \(^{6}\) Division of Human Genetics, School of Pathology, Faculty of Health Sciences, University of the Witwatersrand, Johannesburg, South Africa \(^{7}\) National Health Laboratory Service, Johannesburg, South Africa \(^{8}\) Academy of Science of South Africa  

<|ref|>text<|/ref|><|det|>[[85, 380, 444, 395]]<|/det|>
\*Corresponding author: E- mail: carina.schlebusch@ebc.uu.se.  

<|ref|>text<|/ref|><|det|>[[85, 400, 238, 414]]<|/det|>
Accepted: 12 May 2020  

<|ref|>text<|/ref|><|det|>[[85, 419, 366, 433]]<|/det|>
\(^{\dagger}\) These authors contributed equally to this work.  

<|ref|>text<|/ref|><|det|>[[85, 437, 916, 467]]<|/det|>
Data deposition: The complete Y- chromosome sequences were deposited on the European Genome Phenome Archive (https://www.ebi.ac.uk/ega/), accession number EGAS00001004459, and are available for research use under controlled access policies.  

<|ref|>sub_title<|/ref|><|det|>[[85, 500, 168, 516]]<|/det|>
## Abstract  

<|ref|>text<|/ref|><|det|>[[84, 523, 919, 707]]<|/det|>
Although the human Y chromosome has effectively shown utility in uncovering facets of human evolution and population histories, the ascertainment bias present in early Y- chromosome variant data sets limited the accuracy of diversity and TMRCA estimates obtained from them. The advent of next- generation sequencing, however, has removed this bias and allowed for the discovery of thousands of new variants for use in improving the Y- chromosome phylogeny and computing estimates that are more accurate. Here, we describe the high- coverage sequencing of the whole Y chromosome in a data set of 19 male Khoe- San individuals in comparison with existing whole Y- chromosome sequence data. Due to the increased resolution, we potentially resolve the source of haplogroup B- P70 in the Khoe- San, and reconcile recently published haplogroup A- M51 data with the most recent version of the ISOGG Y- chromosome phylogeny. Our results also improve the positioning of tentatively placed new branches of the ISOGG Y- chromosome phylogeny. The distribution of major Y- chromosome haplogroups in the Khoe- San and other African groups coincide with the emerging picture of African demographic history; with E- M2 linked to the agriculturalist Bantu expansion, E- M35 linked to pastoralist eastern African migrations, B- M112 linked to earlier east- south gene flow, A- M14 linked to shared ancestry with central African rainforest hunter- gatherers, and A- M51 potentially unique to the Khoe- San.  

<|ref|>text<|/ref|><|det|>[[85, 709, 766, 726]]<|/det|>
Key words: Y chromosome, next- generation sequencing, haplogroups, Khoe- San, southern Africa.  

<|ref|>sub_title<|/ref|><|det|>[[85, 755, 202, 771]]<|/det|>
## Introduction  

<|ref|>text<|/ref|><|det|>[[85, 777, 483, 853]]<|/det|>
The male- specific portion of the Y chromosome (MSY) has long been regarded as an effective tool in the study of human evolutionary history (Underhill and Kivisild 2007). It has proved useful mainly due to a lack of recombination along its length, making it the longest  

<|ref|>text<|/ref|><|det|>[[517, 760, 916, 853]]<|/det|>
haplotypic block in the human genome (Scozzari et al. 2012); and its paternal mode of inheritance. The transmission of an intact haplotype from father to son, changing only through mutation, preserves a simpler record of its history and allows us to study the male contribution to the shaping of humanity.